bioimageio/spec/model/v0_4/schema.py

```diff
@@ -479,11 +479,9 @@ def get_min_shape(t) -> numpy.ndarray:
         f"different from `authors` in root or any entry in `weights`.",
     )
 
-    parent = fields.Nested(
-        ModelParent(),
-        bioimageio_description="Parent model from which the trained weights of this model have been derived, e.g. by "
-        "finetuning the weights of this model on a different dataset. For format changes of the same trained model "
-        "checkpoint, see `weights`.",
+    parent = fields.Union(
+        [fields.BioImageIO_ID(resource_type="model"), fields.URI(), fields.RelativeLocalPath()],
+        bioimageio_description="BioImage.IO model id or URL or local relative path to a model RDF",
```

Comment:
Please extend the description, it should be clear what a parent is, i.e. the model from which this model is derived, e.g. by fine-tuning the weights.

Reply:
thanks, I have included that